docs/source/api/plugin/usage-reporting.md

```diff
@@ -125,18 +107,15 @@ By default, all errors get reported to Apollo servers. You can specify a filter
 
 ###### `includeRequest`
 
-`Function`
+`async Function`
 </td>
 <td>
 
-This option allows you to choose if a particular request should be represented in the usage reporting sent to Apollo servers. By default, all requests are included. If this *async* predicate function is specified, its return value will determine whether a given request is included.
-
-The predicate function receives the request context. If validation and parsing of the request succeeds, the function will receive the request context in the [`GraphQLRequestContextDidResolveOperation`](https://www.apollographql.com/docs/apollo-server/integrations/plugins/#didresolveoperation) phase, which permits tracing based on dynamic properties, e.g., HTTP headers or the `operationName` (when available). Otherwise it will receive the request context in the [`GraphQLRequestContextDidEncounterError`](https://www.apollographql.com/docs/apollo-server/integrations/plugins/#didencountererrors) phase. In either case, it should return a `Promise<Boolean>`.
-
-For example, you can look at the GraphQL operation's name in `requestContext.operationName` or at the HTTP headers in `requestContext.request.http?.headers`.
+Specify this asynchronous function to configure which requests are included in usage reports sent to Apollo Studio. For example, you can omit requests that execute a particular operation or requests that include a particular HTTP header.
 
-(If you don't want any usage reporting for any request, don't use this plugin; if you are using other plugins that require you to configure an Apollo API key, use `ApolloServerPluginUsageReportingDisabled` to prevent this plugin from being installed by default.)
+This function is called for each received request. It takes a [`GraphQLRequestContext`](https://github.com/apollographql/apollo-server/blob/main/packages/apollo-server-types/src/index.ts#L115-L150) object and must return a `Promise<Boolean>` that indicates whether to include the request. It's called either after the request is successfully resolved, or after it generates an error.
```

Comment:
after the *operation* is successfully resolved (ie, we've found an operation within the document that we're going to run).

I think your conciseness is good but keeping the link to the exact phase in the plugin doc could still be good.